backend/controller/ingress/request_test.go

```diff
@@ -177,17 +180,53 @@ func TestBuildRequestBody(t *testing.T) {
 			method:    "GET",
 			path:      "/getPath/bob",
 			routePath: "/getPath/{username}",
-			expected: HTTPRequest[PathParameterRequest]{
+			expected: HTTPRequest[ftl.Unit, PathParameterRequest, map[string][]string]{
 				Method: "GET",
 				Path:   "/getPath/bob",
-				PathParameters: map[string]string{
-					"username": "bob",
-				},
-				Body: PathParameterRequest{
+				PathParameters: PathParameterRequest{
 					Username: "bob",
 				},
 			},
 		},
+		{name: "GetById",
+			verb:      "getById",
+			method:    "GET",
+			path:      "/getbyid/100",
+			routePath: "/getbyid/{id}",
+			expected: HTTPRequest[ftl.Unit, int, ftl.Unit]{
+				Method:         "GET",
+				Path:           "/getbyid/100",
+				PathParameters: 100,
+			},
+		},
+		{name: "MapQuery",
+			verb:      "mapQuery",
+			method:    "GET",
+			path:      "/mapQuery",
+			routePath: "/mapQuery",
+			query: map[string][]string{
+				"alias": {"value"},
+			},
+			expected: HTTPRequest[ftl.Unit, ftl.Unit, map[string]string]{
+				Method: "GET",
+				Path:   "/mapQuery",
+				Query:  map[string]string{"alias": "value"},
+			},
+		},
+		{name: "MultiMapQuery",
+			verb:      "multiMapQuery",
+			method:    "GET",
+			path:      "/multiMapQuery",
+			routePath: "/multiMapQuery",
+			query: map[string][]string{
+				"alias": {"value"},
+			},
+			expected: HTTPRequest[ftl.Unit, ftl.Unit, map[string][]string]{
+				Method: "GET",
+				Path:   "/multiMapQuery",
+				Query:  map[string][]string{"alias": []string{"value"}},
+			},
+		},
```

Comment:
Is there a test for mapping query and path parameters onto structs?

Reply:
Yes, there were existing tests that were mapping them to a body struct, these have been changed so now tests are mapping them onto the path or query param struct with a Unit body.